### ISSUE TITLE
senders.NewSender doesn't work for direct ingestion

### ISSUE THREAD

**Issue by keep94** [wavefrontHQ/wavefront-sdk-go#84]
If you call 
```
sender, err := senders.NewSender("https://11111111-2222-3333-4444-555555555555@surf.wavefront.com")
```
To get a sender that does direct ingestion to https://surf.wavefront.com, it won't work because it tries to send to port 2878 instead of to port 443.

If you call
```
sender, err := senders.NewSender("https://11111111-2222-3333-4444-555555555555@surf.wavefront.com:443")
```

This does work, but a user of the API will not think to do this.

This API should be intuitive for users.  Both chunks of code ought to work.

**Comment by LukeWinikates** [comment]
@keep94 does it make sense to close this issue since it has a merged PR tied to it?

**Comment by oppegard** [comment]
Pinging @keep94: was this fixed in #85, so this issue should be closed?
